crypto/aes/asm/aesni-x86_64.pl

```diff
@@ -4710,6 +4725,7 @@ sub aesni_generate8 {
 	shufps	\$0b10101010,%xmm1,%xmm1	# critical path
 	xorps	%xmm1,%xmm2
 	ret
+.cfi_endproc
```

Comment:
Lkey_expansion_256a and key_expansion_256b are something that's called, so maybe they should should have thrown own .cfi_startproc and .cfi_endproc

Reply:
Oh, for added fun, key_expansion_256a_cold also has it's own one.